Composer/packages/client/src/components/AppUpdater.tsx

```diff
@@ -131,6 +131,12 @@ export const AppUpdater: React.FC<{}> = () => {
           break;
         }
 
+        case 'update-in-progress': {
+          setAppUpdateStatus(AppUpdaterStatus.UPDATE_AVAILABLE, undefined);
```

Comment:
the `undefined` argument here is supposed to be the version of the update -- if you look at the first video you posted, you can see that when this line is called and shows the "Update Available" dialog, the dialog shows "vundefined"

![image](https://user-images.githubusercontent.com/3452012/105436889-e241da80-5c14-11eb-86c7-0d45ce471223.png)

Reply:
Nice catch!
I see the differences, now updated: 

![image](https://user-images.githubusercontent.com/49866537/105438796-a4899400-5c9e-11eb-82a4-f84901d825bf.png)